compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -77,9 +77,28 @@ static llvm::cl::opt<bool> clLLVMGPUEnablePrefetch(
 
 llvm::raw_ostream &operator<<(llvm::raw_ostream &os,
                               const LLVMGPUPipelineOptions &options) {
+  if (options.reorderStrategy) {
+    StringRef reorderStr;
+    if (options.reorderStrategy.value() == ReorderWorkgrupsStrategy::Transpose)
+      reorderStr = "transpose";
+    else if (options.reorderStrategy.value() ==
+             ReorderWorkgrupsStrategy::Swizzle)
+      reorderStr = "swizzle";
+    else {
+      assert(options.reorderStrategy.value() ==
+                 ReorderWorkgrupsStrategy::None &&
+             "Unhandled reorder option");
+      reorderStr = "none";
+    }
+
+    return os << "{" << "enableReduceSharedMemoryBankConflicts = "
+              << options.enableReduceSharedMemoryBankConflicts
+              << ", ReorderWorkgroupsStrategy = " << reorderStr
+              << ", enableUkernels = " << options.enableUkernels << "}";
+  }
```

Comment:
We should always print the strategy here. If the value is `std::nullopt` we can print something like `<not set>`